llvm/lib/SYCLLowerIR/PropagateAspectsUsage.cpp

```diff
@@ -0,0 +1,344 @@
+//===---- PropagateAspectsUsage.cpp - PropagateAspectsUsage Pass ----------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// Pass propagates metadata corresponding to usage of optional device
+// features.
+//
+// The pass consists of a four main steps:
```

Comment:
NIT: consists of four main steps

Reply:
Applied in 96139d671baf3ea32672932b73d2dc5c0a56663b